packages/hooks/src/useLiveChat.ts

```diff
@@ -0,0 +1,95 @@
+import { useEffect } from 'react';
+import Cookies from 'js-cookie';
+import { URLUtils } from '@deriv-com/utils';
+import { useIsMounted } from 'usehooks-ts';
+import { useRemoteConfig } from '@deriv/api';
+
+type TLiveChatClientInformation = {
+    is_client_initialized: boolean;
+    is_logged_in: boolean;
+    loginid?: string;
+    landing_company_shortcode?: string;
+    currency?: string;
+    residence?: string;
+    email?: string;
+    first_name?: string;
+    last_name?: string;
+};
+
+const useLiveChat = (client_information: TLiveChatClientInformation) => {
+    const {
+        is_client_initialized,
+        landing_company_shortcode = ' ',
+        currency = ' ',
+        email = ' ',
+        is_logged_in = ' ',
+        loginid = ' ',
+        residence = ' ',
+        last_name = ' ',
+        first_name = ' ',
+    } = client_information;
+
+    const url_query_string = window.location.search;
+    const url_params = new URLSearchParams(url_query_string);
+    const reset_password = URLUtils.getQueryParameter('action') === 'reset_password';
+    const should_disable_livechat = url_params.get('code') && reset_password;
+    const isMounted = useIsMounted();
+
+    const { data } = useRemoteConfig(isMounted());
```

Comment:
@yauheni-deriv Please, don't use `isMounted` here because under the hood it's `ref`, by default it's false, and when it will be changed to true in the own useEffect, it will not trigger component (hook) to repaint, in this case `useRemoteConfig` will not make api request. Described and fixed this issue in this PR (https://github.com/binary-com/deriv-app/pull/16726).